src/App/App.tsx

```diff
@@ -118,6 +120,7 @@ export default function App() {
           path="*"
           element={<Navigate replace to={appRoutes.marketplace} />}
         />
+        <Route path="*" element={<Navigate replace to={appRoutes.home} />} />
```

Comment:
No need to add this, just edit the existing route directly above this one, replacing marketplace with the landing page: https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2894/files#diff-1deef93890e359e3ea9cccd8b9fb992a1008c1f21b9eee105a1a364afae4f170R119-R122